WebRisk/composer.json

```diff
@@ -27,7 +27,7 @@
     },
```

Comment:
Should we be bumping the version of gax here and for WebSecurityScanner as well?